### DISCUSSION TITLE
可以实现透明网关这种方式吗？

### DISCUSSION THREAD

**Discussion by que01** [kkkgo/PaoPaoGateWay#36] [问和答] [answered]
大致的情况是我使用paopao gateway之后，配合内网的dns处理好```paopao.dns:7889/ppgw.ini```之后，使用浏览器插件设置```socks5://ip:1080```进行代理后非常顺利一次性就成功了。

但是现在问题是我前期其实是有一个openwrt在内网实现了一个透明网关，我可以直接在路由器上进行配置特定设备走openwrt的ip作为网关，这样设备就可以直接科学不需要设备端配置了。所以我想问一下，我在使用suburl模式的前提下，是否有可能实现类似的功能呢？这样我就可以关掉这个openwrt了。

以上
&ensp; &ensp; 祝好。

**Comment by kkkgo** [comment, marked as answer]
这个本来就是当透明网关用的，socks5反而是备用……
ppgw的用法是fakeip网关，路由器上添加一条静态路由就可以自动分流，可以参考文档里面的 与DNS服务器配合完成分流 一节，或者参考其他人分享的配置：https://github.com/kkkgo/PaoPaoGateWay/discussions/categories/%E9%85%8D%E7%BD%AE%E5%88%86%E4%BA%AB

**Reply by kkkgo** [reply]
参考拓扑 [→详细说明《FakeIP网关的工作原理》](https://blog.03k.org/post/paopaogateway.html)

**Comment by que01** [comment]
加上paopaodns解决了 不过为了使用adghome做统计 将ppdns设为了唯一上游dns
